packages/web-config-server/src/__tests__/permissions.test.js

```diff
@@ -0,0 +1,181 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { createApp } from '/app';
+import { TestableApp } from './TestableApp';
+
+const accessPolicy = {
+  permissions: {
+    reports: {
+      _items: {
+        DL: {
+          _access: {
+            Public: true,
+          },
+          _items: {
+            DL_North: {
+              _access: {
+                Admin: true,
+                Donor: true,
+                'Royal Australasian College of Surgeons': true,
+              },
+              _items: {
+                DL_North_Slytherin: {
+                  _access: {
+                    Public: false,
+                  },
+                },
+              },
+            },
+            'DL_South West': {
+              _access: {
+                Public: false,
+              },
+            },
+            'DL_South East': {
+              _items: {
+                'DL_South East_Gryffindor': {
+                  _items: {
+                    DL_3: {
+                      _access: {
+                        Admin: true,
+                        Donor: true,
+                        'Royal Australasian College of Surgeons': true,
+                      },
+                    },
+                  },
+                },
+              },
+            },
+          },
+        },
+        TO: {
+          _access: {
+            Public: true,
+          },
+          _items: {
+            TO_Tongatapu: {
+              _access: {
+                Admin: true,
+                Donor: true,
+                'Royal Australasian College of Surgeons': true,
+              },
+            },
+          },
+        },
+      },
+    },
+  },
+};
+
+describe.skip('UserHasAccess', () => {
+  let app;
+
+  beforeAll(async () => {
+    app = new TestableApp(await createApp());
```

Comment:
I think that `createApp()` needs not be async: [app.js](https://github.com/beyondessential/tupaia/blob/dev/packages/web-config-server/src/app.js)

It is not async in `central-server`

Reply:
Yep, doesn't need async, can you remove it @kael89? 🙏

Reply:
Happy to do so in another PR though! (Don't want touch production code in this one)